kw

```diff
@@ -22,9 +22,10 @@ KW_ETC_DIR="${XDG_CONFIG_HOME:-"${HOME}/.config"}/${KWORKFLOW}"
 # Set system-wide path
 if [[ -f "$KW_SYSTEM_WIDE_INSTALLATION" ]]; then
   KW_LIB_DIR='/usr/share/kw'
+  KW_DB_DIR="/usr/share/kw/database"
   KW_SOUND_DIR='/usr/share/sounds/kw'
   KW_DOC_DIR='/usr/share/doc/kw/html/'
-  KW_SOUND_DIR='/usr/share/sounds/kw'
+  KW_MAN_DIR='/usr/share/man/en/kw'
```

Comment:
Are you sure this path is correct?

Reply:
More or less sure. Debian keeps man pages at `/usr/share/` under their language abbreviation, such as: jp, ch, es, pt. That is why I put it in `/usr/share/man/en/kw` instead of `/usr/share/man/kw`.

Reply:
Tbh, I'm a little bit confused about the man path.

I checked this path in my Debian and Arch Linux systems, and none have the subfolder `en` (my system is set to English). I'm under the impression that the default English documentation for a command publicly available for the system should be in the subfolder `man1`. However, I'm not 100% sure... let's try to be sure about this.